src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntity.java

```diff
@@ -1183,4 +1185,9 @@ public Timing getTimingsHandler() {
         }
         return this.timing;
     }
+
+    @Override
+    public boolean hasGravity() {
+        return this.get(Keys.ENTITY_HAS_GRAVITY).get();
+    }
```

Comment:
Well then I would need to change an interface, because there is no method to override that returns a `SpongeValue`.